library/kani_macros/src/contracts.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright Kani Contributors
```

Comment:
This should be left to a different PR?

Reply:
Which part? The copyright information?

Reply:
No, I meant this whole file. This seems to be for adding the Kani macros for pre-post conditions? If so, should this be in a separate PR?